File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
from collections import Counter
def solution(s):
    answer = []
    cnt_zero = 0
    cnt_result = 0
    while s != "1":
        cnt_list = Counter(s)
        cnt = len(s) - cnt_list["0"]
        tmp_bin = bin(cnt)[2:]
        s = tmp_bin
        cnt_result += 1
        cnt_zero += cnt_list["0"]
    answer.append(cnt_result)
    answer.append(cnt_zero)
    return answer

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    var answer = [];
    var cnt_result = 0;
    var cnt = 0
    while (s !== "1"){
        var cnt_zero = 0;
        for (let j=0; j<s.length; j++) {
            if (s[j] === "0") {
                cnt_zero = cnt_zero + 1
            }
        }
        tmp = (s.length - cnt_zero).toString(2)
        cnt_result = cnt_result + 1
        cnt = cnt + cnt_zero
        s = tmp
    }
    answer.push(cnt_result);
    answer.push(cnt);
    return answer;
}